AntennaTracker/Tracker.h

```diff
@@ -210,6 +210,7 @@ class Tracker : public AP_Vehicle {
     void update_yaw_cr_servo(float yaw);
 
     // system.cpp
+    ModeReason control_mode_reason = ModeReason::UNKNOWN;
```

Comment:
If I understood it correctly the Idea would be to have this variable in the AP_Vehicle class, and not here.

Reply:
On DevCallEU on 27.01.2021@tridge told me to make it virtual like e.g. get_wp_dist_m().
And I think that's wise because in that way we have an associated block of code with set_mode(), get_mode() and get_mode_reason().

If we would define it in the base class with a pair of accessors we had to touch many lines of code and, as far as I can see, without a clear advantage.
![2021-01-30 11_23_20-Window](https://user-images.githubusercontent.com/39805388/106387200-f89c2300-63d8-11eb-89fb-042a9b3cd220.png)